graylog2-server/src/main/resources/swagger/lib/swagger.js

```diff
@@ -1004,10 +1004,15 @@
       }
       responseContentType = null;
       if (this.type === "POST" || this.type === "GET" || this.type === "PATCH") {
+        responseContentType = "application/json";
```

Comment:
Couldn't this be pulled into line 1005? The else branch overwrites it with `null` anyway, so this could simply be the initial value of `responseContentType`.